web/modules/custom/joinup_communities/tallinn/src/Plugin/Field/FieldWidget/TallinnEntryWidget.php

```diff
@@ -35,7 +35,12 @@ class TallinnEntryWidget extends WidgetBase {
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $item = $items[$delta];
 
-    $element['#type'] = 'fieldset';
+    $wrapper_classes = 'js-form-wrapper form-wrapper';
+    $element = [
+      '#type' => 'details',
+      '#title' => $this->fieldDefinition->getLabel() . ' - ' . $this->fieldDefinition->getDescription(),
```

Comment:
Ok, so this has been a really bad merge. After you updated 4383 I merged in and quickly solved the conflicts but apparently I did it in a bad way.

The first one, is out of the question. Was not there last time I checked and I removed the description as well as it should indeed not be there.

The second, the error, is the reason in the end why I switched from a static validation method to a normal one. Because I was able to easily use the `$this->fieldDefinition->getLabel()` and get the label. I have added an extra property in order to use it in that case, called `#label`. This way, there is no reason to re load the definition.